ts/models/conversation.ts

```diff
@@ -1490,9 +1542,7 @@ export class ConversationModel extends Backbone.Model<ConversationAttributes> {
     ) {
       return false;
     }
-    const msgRequestsEnabled =
-      window.lokiFeatureFlags.useMessageRequests &&
-      window.inboxStore?.getState().userConfig.messageRequests;
+    const msgRequestsEnabled = window.inboxStore?.getState().userConfig.messageRequests;
```

Comment:
you can remove that test and just return   `return Boolean(this.get('isApproved'));`